wandb/filesync/dir_watcher.py

```diff
@@ -291,18 +299,14 @@ def _get_file_event_handler(
                 )
             else:
                 make_handler = PolicyEnd
-                for policy, globs in self._user_file_policies.items():
-                    if policy == "end":
-                        continue
+                for policy, handler_type in [("now", PolicyNow), ("live", PolicyLive)]:
+                    globs = self._user_file_policies[policy]
                     # Convert set to list to avoid RuntimeError's
                     # TODO: we may need to add locks
                     for g in list(globs):
```

Comment:
```
for g in list(self._user_file_policies[policy]):
```
total nit

Reply:
(obsolete because I reverted almost all the changes to the business logic, sorry to have made you review them)